staging/src/k8s.io/apiserver/pkg/server/dynamiccertificates/named_certificates_test.go

```diff
@@ -225,8 +226,8 @@ NextTest:
 		c := DynamicServingCertificateController{sniCerts: sniCerts}
 		content, err := c.newTLSContent()
 		assert.NoError(t, err)
-
-		certMap, err := c.BuildNamedCertificates(content.sniCerts)
+		ctx := context.TODO()
```

Comment:
In tests, use `ktesting.NewTestContext` to construct a ctx with per-unit output.